asab/config.py

```diff
@@ -174,6 +188,51 @@ def _load(self):
 
 		del self._load_dir_stack
 
+	def _from_include_build_url(self, purl):
+
+		parse_include = urlparse(purl)
+		url_path = parse_include.path
+		url_scheme = parse_include.scheme
+		url_netloc = self["asab.zookeeper"]["url"]
+
+		if purl.rfind("///") == 10:
+			url_comp = (url_scheme, url_netloc, url_path, '', '', '')
+
+		elif purl.rfind(":.") == 9:
+			new_url_path = self["asab.zookeeper"]["path"] + url_path[1:]
+			url_comp = (url_scheme, url_netloc, new_url_path, '', '', '')
+
+		#unparse  components into proper URL
+		url_built = urlunparse(url_comp)
+		return  url_built
+
+	def _include_from_zookeeper(self, zkurl):
+		import aiozk
+
+		loop = asyncio.get_event_loop()
+		#parse include value into hostname and path
+		url_pieces = urlparse(zkurl)
+		url_path = url_pieces.path
+		url_netloc = url_pieces.netloc
+
+		async def download_from_zookeeper():
+			try:
+				zk = aiozk.ZKClient(
+					url_netloc,
+					allow_read_only=True,
+					read_timeout=60,  # seconds #
+			   	)
+				await zk.start()
+				data = await zk.get_data(url_path)
+				#convert bytes to string
+				encode_config = str(data,'utf-8')
+				self.read_string(encode_config)
+				await zk.close()
+			except Exception as e:
+				L.warning("Connection to zookeeper could not be established: '{}'.".format(e))
```

Comment:
"Failed to obtain configuration from zookeeper server(s):" - and it is error, not warning.